app/models/post_timing.rb

```diff
@@ -58,8 +58,8 @@ def self.record_timing(args)
     record_new_timing(args) if rows == 0
   end
 
-  def self.destroy_last_for(user, topic_id)
-    topic = Topic.find(topic_id)
+  def self.destroy_last_for(user, topic_id, topic: nil)
```

Comment:
Not a super fan of the signature of this method. Maybe there should be two methods instead? One accepting a topic_id, another one accepting a topic object. And the one accepting the topic_id would find the topic and then call the other method?

Reply:
I wasn't either, but I did not want to have another method for this. We already have two: destroy_for and destroy_last_for. I changed this a bit, is it better now? If it's not much better, I will write another method.

Reply:
I guess that works too 👍